# Speed command for A1. Right now only vel_x, vel_y and rot_z are supported.

float32 vel_x
float32 vel_y
float32 rot_z
